src/js/pensions/components/EmploymentField.jsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import moment from 'moment';
+
+export default function FullNameField({ formData }) {
```

Comment:
Component name doesn't match file name. I think you meant to change this to `EmploymentField` after copying. 😉